aten/src/ATen/core/jit_type.h

```diff
@@ -519,7 +519,8 @@ struct CAFFE2_API TensorType : public Type {
       const VaryingShape<int64_t>& sizes,
       const VaryingShape<int64_t>& strides,
       c10::optional<bool> requires_grad,
-      c10::optional<bool> undefined = false);
+      c10::optional<bool> undefined = false,
+      bool tensor_continuity = false);
```

Comment:
I think they should be named `tensor_contiguity`

Reply:
lol thanks for catching it!